py/picca/delta_extraction/data_catalogues/desi_healpix.py

```diff
@@ -125,58 +129,69 @@ def read_data(self):
 
         is_sv = True
         is_mock = False
+        forests_by_targetid = {}
+
+        arguments = []
+        for (healpix, survey), group in zip(grouped_catalogue.groups.keys,
```

Comment:
We don't need to zip grouped_catalogue, but if that's what you prefer that's fine. Not sure if there is an overhead with zipping.

Reply:
I added them with the `zip` as I didn't like accessing the first element in the group to get them, but then there will always be at least an element on each group (otherwise the `(healpix, survey)` pair does now appear). I changed it back to how you did it in your version.